ios/PurchasesHybridCommon/PurchasesHybridCommon/EntitlementInfo+HybridAdditions.swift

```diff
@@ -26,8 +26,8 @@ import RevenueCat
             "store": storeString,
             "productIdentifier": productIdentifier,
             "isSandbox": isSandbox,
-            "unsubscribedDetectedAt": unsubscribeDetectedAt?.rc_formattedAsISO8601() ?? NSNull(),
-            "unsubscribedDetectedAtMillis": unsubscribeDetectedAt?.rc_millisecondsSince1970AsDouble() ?? NSNull(),
+            "unsubscribeDetectedAt": unsubscribeDetectedAt?.rc_formattedAsISO8601() ?? NSNull(),
+            "unsubscribeDetectedAtMillis": unsubscribeDetectedAt?.rc_millisecondsSince1970AsDouble() ?? NSNull(),
```

Comment:
This is a breaking change for React Native and Cordova, since these get transformed directly into Javascript objects. We have typescript definitions, but using typescript is not mandatory. Should we maybe return both this wrong version and the correct one?

What do you think?

Reply:
Oh so it wasn't working at all before?

Reply:
Nope. That's what I meant by the need for end-to-end tests so we can catch this stuff.

Reply:
Our React Native and Cordova plugins just provide Typescript interfaces, we don't do any parsing. This will break anyone that's not using typescript and instead looked at the Javascript object directly, saw we were sending 'unsubscribed' and is accessing that key in the JavaScript object we send down.

It is not very probable anyone is doing that and probably safe to change, but wanted to point out this breaks whoever is accessing that key directly.

In the past, I've kept both versions for a while just in case

Reply:
> It is not very probable anyone is doing that and probably safe to change, but wanted to point out this breaks whoever is accessing that key directly.

Oooh I see what you're saying.
We could offer both, but given that those users are using a key that wasn't documented, maybe it's fine. What do you think?

Reply:
Sure, makes sense. Let's ship it!

Reply:
Thanks!